osm2streets/src/render.rs

```diff
@@ -369,10 +389,17 @@ impl StreetNetwork {
 
         let mut pairs = Vec::new();
         for ((from, _), polygon) in self.movements_for_intersection(i) {
-            if from == road.id {
+            if from == id {
                 pairs.push((polygon.to_geojson(Some(&self.gps_bounds)), make_props(&[])));
             }
         }
+        if pairs.is_empty() {
```

Comment:
In general, when should we expect turns to exist, but not movements? Right now `update_i` does both. Is this temporary before we have movements defined in all cases?

Reply:
Yes, that was a temporary measure. I don't know where it will land though. I think we will be able to calculate movements any time we calculate turns, so we will be able to remove the `Option`.

I was playing with using an `Option` in places where data is only present after some step of the algorithm, instead of using `*::dummy()` kind of values.